### PR TITLE
Add options to hide label and disable link

### PR DESCRIPTION
**Pull request by laloptk**
[KAD-2360](https://stellarwp.atlassian.net/browse/KAD-2360)

When disableing links, the href attribute is not printed because it is not required for a tags as stated here: https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-href

[KAD-2360]: https://stellarwp.atlassian.net/browse/KAD-2360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by mark-c-woodard** [comment]
Resolve the merge conflict and get the latest from nav.

File: src/blocks/navigation-link/edit.js [hunk 1]
```diff
@@ -994,7 +1006,7 @@ export default function Edit(props) {
 							onChange={(relValue) => {
 								setAttributes({ rel: relValue });
 							}}
-							label={__('Rel attribute')}
+							label={__('Rel attributte')}
```
[the remark this answers is not in the document]
Comment (laloptk) [reply src/blocks/navigation-link/edit.js:1011]: Done!

File: includes/blocks/class-kadence-blocks-navigation-link-block.php [hunk 2]
```diff
@@ -280,7 +279,7 @@ public function build_html( $attributes, $unique_id, $content, $block_instance )
 		$icon  = ! empty( $svg_icon ) ? '<div class="link-media-container"><span class="link-svg-icon link-svg-icon-' . esc_attr( $nav_link_attributes['mediaIcon'][0]['icon'] ) . '">' . $svg_icon . '</span></div>' : '';
 
 		return sprintf(
-			'<li %1$s><div class="link-drop-wrap"><a class="wp-block-kadence-navigation-link__content" href="' . esc_url( $url ) . '"><span class="link-drop-title-wrap">' . esc_html( $label ) . $icon . '<span class="title-dropdown-navigation-toggle">%2$s</span></span></a></div>%3$s</li>',
+			'<li %1$s><div class="link-drop-wrap"><a class="wp-block-kadence-navigation-link__content"' . (! empty($nav_link_attributes['disableLink']) && true === $nav_link_attributes['disableLink'] ? '' : ' href="' . esc_url( $url ) . '"' ) . '><span class="link-drop-title-wrap">' . ( ! empty( $nav_link_attributes['hideLabel'] ) && true === $nav_link_attributes['hideLabel'] ? '' : esc_html( $label ) ) . $icon . '<span class="title-dropdown-navigation-toggle">%2$s</span></span></a></div>%3$s</li>',
```
[the remark this answers is not in the document]
Comment (laloptk) [reply includes/blocks/class-kadence-blocks-navigation-link-block.php]: Done!

File: src/blocks/navigation-link/edit.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (laloptk) [reply src/blocks/navigation-link/edit.js]: Done!

File: src/blocks/navigation-link/edit.js [hunk 3]
```diff
@@ -1716,7 +1728,7 @@ export default function Edit(props) {
 												ref={ref}
 												identifier="label"
 												className="wp-block-navigation-item__label"
-												value={label}
+												value={hideLabel ? '' : label}
```
[the remark this answers is not in the document]
Comment (laloptk) [reply src/blocks/navigation-link/edit.js:1733]: Done!

**Pull request merged**